pkg/agent/cniserver/pod_configuration.go

```diff
@@ -327,6 +165,10 @@ func ParseOVSPortInterfaceConfig(portData *ovsconfig.OVSPortData, portConfig *in
 	return interfaceConfig
 }
 
+func (pc *podConfigurator) initialize() error {
```

Comment:
Maybe merge it into newPodConfigurator()?

Reply:
This function is called in cniServer.Initialze(), and we could return errors in this phase.